docs/install_guides/installing-openvino-docker-windows.md

```diff
@@ -27,148 +29,227 @@ This guide provides steps for creating a Docker image with Intel® Distribution
 To use GPU Acceleration in Windows containers, make sure that the following requirements for Windows host, OpenVINO and Docker are met:
 
 - [Windows requirements](https://docs.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/gpu-acceleration):
-  - The container host must be running Windows Server 2019 or Windows 10 of version 1809 or higher.
-  - The container base image must be `mcr.microsoft.com/windows:1809` or higher. Windows Server Core and Nano Server container images are not currently supported.
-  - The container host must be running Docker Engine 19.03 or higher.
-  - The container host must have GPU running display drivers of version WDDM 2.5 or higher.
-- GPU requirement for OpenVINO: Intel Graphics Driver for Windows of version 15.65 or higher.
+   - The container host must be running Windows Server 2019 or Windows 10 of version 1809 or higher.
+   - The container base image must be `mcr.microsoft.com/windows:1809` or higher. Windows Server Core and Nano Server container images are not currently supported.
+   - The container host must be running Docker Engine 19.03 or higher.
+   - The container host must have GPU running display drivers of version WDDM 2.5 or higher.
+- GPU requirement for OpenVINO: 
+   - Intel Graphics Driver for Windows of version 15.65 or higher.
 - [Docker isolation mode requirements](https://docs.microsoft.com/en-us/virtualization/windowscontainers/manage-containers/hyperv-container):
-  - Windows host and container version tags must match.
-  - [Windows host and container isolation process support](https://docs.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/version-compatibility).
+   - Windows host and container version tags must match.
+   - [Windows host and container isolation process support](https://docs.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/version-compatibility).
 
 ## Installation Flow
```

Comment:
Changes starting from here need be reverted. We agreed on having a simple overview of the flow first and then the detailed steps.